news/2022-01-14-taiko-mania-pp-sr-survey.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: "osu!taiko and osu!mania Performance Points & Star Rating Survey"
+date: 2022-01-14 17:00:00 +0000
+---
+
+A call to arms for players from the osu!taiko and osu!mania game modes! We want your feedback on the way performance points and star rating development should proceed - read on to get involved!
+
+![](/wiki/shared/news/2022-01-14-taiko-mania-pp-sr-survey/star-rating.jpg)
+
+Six months ago, the osu!team released a [public survey](https://osu.ppy.sh/home/news/2021-08-17-pp-sr-survey) to assess the community's position on both current and future deployments of performance and difficulty calculation. The [results of the survey](https://osu.ppy.sh/home/news/2021-09-02-pp-sr-survey-results) helped us better understand the community's expectations, as well as determine the best path moving forward.
+
+For the reasons above, we are now opening surveys for the **osu!taiko** and **osu!mania** game modes.
+
+The link to the survey will require you to authenticate your osu! profile - please do not change any of the pre-populated fields on the first page of the survey, otherwise your response may not be recorded.
```

Comment:
```suggestion
The link to the survey will require you to authenticate your osu! profile — please do not change any of the pre-populated fields on the first page of the survey, otherwise your response may not be recorded.
```